### ISSUE TITLE
strange loss

### ISSUE THREAD

**Issue by ZhuangLii** [cvlab-yonsei/RRID#2]
HI, 
 RRID is  an excellent work, I tried to apply the RRID to train on my own datasets, but I got strange loss curve , and it seems difficult to converge.
![image](https://user-images.githubusercontent.com/47266378/96847151-6acda500-1485-11eb-8273-6c2f37367269.png)
my config
`
lr : 0.01
lr_scheduler : multistep
ID loss: arcface
Triplet loss: semi hard
optimizer: Ranger
`
could you please provide me some advise
THANKS for your time

**Comment by mohuang1996** [comment]
hi！
Did you reproduce the author's code? What is the map value of market1501 that you reproduced? I retrained according to the code given by the author and the result was lower than in the paper. The following is the result of my repetition:

Mean AP: 86.6%
CMC Scores  market1501
  top-1          94.2%
  top-5          98.0%
  top-10         98.7%

I don't know what went wrong.....

**Comment by ZhuangLii** [comment]
> hi！
> Did you reproduce the author's code? What is the map value of market1501 that you reproduced? I retrained according to the code given by the author and the result was lower than in the paper. The following is the result of my repetition:
> 
> Mean AP: 86.6%
> CMC Scores market1501
> top-1 94.2%
> top-5 98.0%
> top-10 98.7%
> 
> I don't know what went wrong.....

你是直接下git clone 复现的吗？我是在别的数据集上测试的，效果不好

**Comment by mohuang1996** [comment]
是的，我是想着先复现下作者提供的数据集在看看其他的。你复现了哪个数据集？效果如何呢？------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;&quot;ZhuangLii&quot;<notifications@github.com&gt;
发送时间:&nbsp;2020年11月3日(星期二) 晚上7:09
收件人:&nbsp;&quot;cvlab-yonsei/RRID&quot;<RRID@noreply.github.com&gt;;
抄送:&nbsp;&quot;mohuang1996&quot;<1722325997@qq.com&gt;;&quot;Comment&quot;<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [cvlab-yonsei/RRID] strange loss (#2)

**Comment by ZhuangLii** [comment]
> 是的，我是想着先复现下作者提供的数据集在看看其他的。你复现了哪个数据集？效果如何呢？------------------&nbsp;原始邮件&nbsp;------------------ 发件人:&nbsp;&quot;ZhuangLii&quot;<notifications@github.com&gt; 发送时间:&nbsp;2020年11月3日(星期二) 晚上7:09 收件人:&nbsp;&quot;cvlab-yonsei/RRID&quot;<RRID@noreply.github.com&gt;; 抄送:&nbsp;&quot;mohuang1996&quot;<1722325997@qq.com&gt;;&quot;Comment&quot;<comment@noreply.github.com&gt;; 主题:&nbsp;Re: [cvlab-yonsei/RRID] strange loss (#2)

是一个比赛，貌似这个不好调节
